.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Declare files that will always have CRLF line endings on checkout.
+infrastructure/docker/services/frontend/etc/service/*/* text eol=lf
```

Comment:
all files should be lf :) 
I always use [this config](https://github.com/lyrixx/dotfiles/blob/master/.gitconfig#L40-L41)